code/datums/datacore.dm

```diff
@@ -95,6 +96,28 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
 			crimes |= crime
 			return
 
+/**
+ * Marks a crime as served for on the security record.
+ *
+ * Is used change the status of a single crime between "Served" and "Not Served" someone's security record.
+ * Arguments:
+ * * id - record id.
+ * * cDataId - id of already existing crime.
+ */
+/datum/datacore/proc/serveCrime(id, cDataId)
+	for(var/datum/data/record/R in security)
```

Comment:
needs better var names and you can use `as anything` here

Reply:
Is the new name okay?